backend/src/main/java/pl/edu/pw/ee/backend/api/order/data/OrderDataDTO.java

```diff
@@ -3,6 +3,6 @@
 import lombok.Builder;
 
 @Builder
-public record OrderDataDTO(String city, int clientId, String email, String name, String orderDate, int orderId,
-                            String phoneNumber, float price, String street, String surname, String zipCode) {
+public record OrderDataDTO(String city, long clientId, String email, String name, String orderDate, int orderId,
+                            String phoneNumber, float price, String street, String surname, String zipCode, String comment) {
```

Comment:
tego `comment` nie było w modelu danych, czy ja zgubiłem? Jeżeli nie było tego u mnie to trzeba by spingować architektów tutaj